.github/workflows/schema.yml

```diff
@@ -25,7 +25,20 @@ jobs:
           with:
             pixi-version: v0.15.2
```

Comment:
```suggestion
            pixi-version: latest
```

Reply:
You can remove this line altogether as `latest` is the default. Also, `cache: true` is also the default.